### ISSUE TITLE
Make Source Available to Allow Fork

### ISSUE THREAD

**Issue by TheGoodIdeaFairy** [TangoTek/TekTopia-Community#1074]
**Describe the bug**
There is a potentially thriving community of coders here interested in improving what is at it's core a solid mod.  It does a disservice to the dev's vision for this to die on the vine.

**To Reproduce**
As above:
- Make source available.
- Allow fork.
- Promote dev admins to manage in original dev's stead.

**Expected behavior**
Bug fixes, new features, compatibility modules.

**Comment by mandar1jn** [comment]
Laughs in owning the source and a working build configuration without the consent of TangoTek. I am currently working on fixing lots of issues with the mod. Once I  truly have something to show I will ask him if I can publish a new version and maybe the source.

**Comment by TheGoodIdeaFairy** [comment]
My biggest wishlist item is BWM compatibility, especially with the Hardcore feature suite (Blacksmiths, Lumberjacks, Farmers, and Chefs are completely borked without turning off a bunch of settings).  That and multiple/localized storage rooms.  Godspeed mandar1jn.

**Comment by Boy132** [comment]
> Laughs in owning the source and a working build configuration without the consent of TangoTek. I am currently working on fixing lots of issues with the mod. Once I truly have something to show I will ask him if I can publish a new version and maybe the source.

Good luck. Some others tried before (including me) and you'll either get no reply or a "no". :(

**Comment by MagmaSlime123** [comment]
Mod's been dead for a year or so now due to Tango being too busy with the 500 other things he's doing. Really shouldn't have made this tbh.

**Comment by Novabella** [comment]
> Laughs in owning the source and a working build configuration without the consent of TangoTek. I am currently working on fixing lots of issues with the mod. Once I truly have something to show I will ask him if I can publish a new version and maybe the source.

Just don't ask, he'll never say yes. He's like a hoarder with his code. He acts like he cares about his project but he hasn't touched it in like two years now. Maybe more I can't remember. Tango doesn't care about tektopia, he just cares that it's his.

**Comment by Novabella** [comment]
> > Laughs in owning the source and a working build configuration without the consent of TangoTek. I am currently working on fixing lots of issues with the mod. Once I truly have something to show I will ask him if I can publish a new version and maybe the source.
> 
> Just don't ask, he'll never say yes. He's like a hoarder with his code. He acts like he cares about his project but he hasn't touched it in like two years now. Maybe more I can't remember. Tango doesn't care about tektopia, he just cares that it's his.

I say this as a firm believer of death to the author

**Comment by MagmaSlime123** [comment]
Better off just taking the idea and making a completely new mod.

Same with his other ideas. 1.14x did add better-**ish** villagers, but they're terribly unbalanced.

**Comment by mandar1jn** [comment]
Yeah. I asked him a long time ago and was very explicit in not wanting me to use any of his code or assets

**Comment by mandar1jn** [comment]
Yeah. I asked him a long time ago and was very explicit in not wanting me to use any of his code or assets

**Comment by smmmadden** [comment]
sadly, he obviously had zero plans after releasing it to support it in anyway.  Why bother even having this site up to report issues, when nothing gets fixed?  This gives a false impression that work is still being done for the mod and we clearly know that is not the case.  Quite disappointed actually as it had a lot of promise!

**Comment by MagmaSlime123** [comment]
> sadly, he obviously had zero plans after releasing it to support it in anyway. Why bother even having this site up to report issues, when nothing gets fixed? This gives a false impression that work is still being done for the mod and we clearly know that is not the case. Quite disappointed actually as it had a lot of promise!

Agreed.

**Comment by mandar1jn** [comment]
Yeah. It’s a mod with a lot of potential

**Comment by MagmaSlime123** [comment]
> Yeah. It’s a mod with a lot of potential

I doubt your a mod developer, but it may be worth re-doing his projects n' such.

**Comment by mandar1jn** [comment]
As a matter of fact I am. But I’m currently working on a big hollow knight mod called Fyremoth. I might give it a shot after that

**Comment by mandar1jn** [comment]
@MagmaBro123 are you in the tektopia discord? If so: what is your username#number or your id? I have decided to start newtopia

**Comment by Novabella** [comment]
> @MagmaBro123 are you in the tektopia discord? If so: what is your username#number or your id? I have decided to start newtopia

Yo if you need an artist for the villager skins I'd be willing to give that a shot.

**Comment by mandar1jn** [comment]
@Novabella great. @MagmaBro123 could you both dm me on discord? (mandar1jn#0990) I'm in the tektopia discord

**Comment by Novabella** [comment]
You're missing a number on that

**Comment by mandar1jn** [comment]
@Novabella yeah. It’s mandar1jn#0990

**Comment by mandar1jn** [comment]
0990 why does it remove the first 0

**Comment by Novabella** [comment]
> 0990 why does it remove the first 0

Sent

**Comment by MagmaSlime123** [comment]
> @MagmaBro123 are you in the tektopia discord? If so: what is your username#number or your id? I have decided to start newtopia

I'm on there but never talk.

@MagmSlime_123#4333, btw. (Friend Requests are completely off so I'm not spammed by bots, just ping me.)
